protoc_plugin/lib/bazel.dart

```diff
@@ -43,7 +43,7 @@ class BazelOptionParser implements SingleOptionParser {
   BazelOptionParser(this.output);
 
   @override
-  void parse(String name, String value, onError(String message)) {
+  void parse(String name, String value, Function(String message) onError) {
```

Comment:
Can you use the `OnError` typedef defined in `options.dart`?

Reply:
You're right